library/src-non-bootstrapped/scala/tasty/reflect/QuotedOps.scala

```diff
@@ -22,7 +24,9 @@ trait QuotedOps extends Core { self: Printers =>
     def unseal(implicit ctx: Context): TypeTree =
       kernel.QuotedType_unseal(tpe)
 
-    /** Show a source code like representation of this type */
+    /** Show a source code like representation of this type
+     *  Will print Ansi colors if ctx.printColors is enabled.
+     */
     def show(implicit ctx: Context): String =
       unseal.showCode
```

Comment:
It's better to default to no colors other than `ctx.printColors` as that's what users want most of the time.

Otherwise, users will be forced to write `x.show(the[Context].withColors)` instead of `x.show`.

Reply:
We can do either with a default `color: Boolean = false` or have two methods:

- show
- showColored

Both are more friendly than `x.show(the[Context].withColors)`

Reply:
We cannot a default the is different for the compiler setting or every other use case will break.

As shown in the example it is easy to add your own version of `show` where you disregard the default settings.

Reply:
We can always add some variant later

Reply:
I think it's better to have it now as we go -- we will forget all these minor usability issues. If such usability issues accumulate, the user experience with Tasty Reflect will degrade.